looker/looker-core-psc/main.tf

```diff
@@ -0,0 +1,36 @@
+ /**
+ * Copyright 2025 Google LLC
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *      http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+# [START looker_google_looker_instance_psc]
+# Create an ENTERPRISE edition Looker (Google Cloud core) instance that has PSC enabled.
+resource "google_looker_instance" "default" {
+  name               = "my-instance"
+  platform_edition   = "LOOKER_CORE_ENTERPRISE_ANNUAL"
+  region             = "us-central1"
+  private_ip_enabled = false
+  public_ip_enabled  = false
+  psc_enabled        = true
+  oauth_config {
+    client_id     = "my-client-id"
+    client_secret = "my-client-secret"
+  }
+  psc_config {
+    allowed_vpcs = ["projects/test-project/global/networks/test"]
+    # (Optional) List of VPCs that are allowed ingress into the Looker instance.
+    service_attachments = [{local_fqdn: "www.local-fqdn.com" target_service_attachment_uri: "projects/my-project/regions/us-east1/serviceAttachments/sa"}]
```

Comment:
If I'm reading the docs correctly, you don't set these values, they are updated for you. Trying to validate this part of the Terraform is failing with `An argument named "service_attachments" is not expected here.`. Are you able to confirm this is intended as an argument?

Reply:
Ah, I get it! So the service attachment argument is relevant only when updating an instance config, not when creating it. I think that to match the other blocks in https://cloud.google.com/looker/docs/looker-core-create-psc#create_instance we should remove that argument entirely.

As for the `allowed_vpcs`, you would only specify that if you are creating an instance that uses only private IP (that is:

```
private_ip_enabled = true
public_ip_enabled  = false
psc_enabled  = true
```
This example doesn't follow that pattern, so I have commented out the `allowed_vpcs` and added some context.